photon-ml/src/main/scala/com/linkedin/photon/ml/util/PalDBIndexMap.scala

```diff
@@ -173,12 +175,18 @@ object PalDBIndexMap {
     * @param partitionId the partition Id
     * @return the formatted filename
     */
-  def partitionFilename(partitionId: Int, namespace: String = IndexMap.GLOBAL_NS): String =
-    s"paldb-partition-${namespace}-${partitionId}.dat"
+  def partitionFilename(
+    partitionId: Int,
+    namespace: String = IndexMap.GLOBAL_NS): String = {
 
-  class PalDBIndexMapIterator(private val indexMap: PalDBIndexMap) extends Iterator[(String, Int)] {
-    private var _iter: Iterator[JMap.Entry[Any, Any]] = null
-    private var _currentItem: (String, Int) = null
+    s"paldb-partition-$namespace-$partitionId.dat"
+  }
```

Comment:
The previous way is correct according to the Scala style guide ([here specifically](http://docs.scala-lang.org/style/declarations.html#body)).